templates/docs/patterns/notification/index.md

```diff
@@ -86,7 +86,7 @@ View example of the time notification pattern
 
 ## Class reference
 
-{% include "_class-references/notification.html" %}
+{{ class_reference() }}
```

Comment:
Why is this one missing `"notification"` in the brackets?

Reply:
Doesn't need it. The function is smart enough to match it from the page it is on (via URL), only needs it when URL slug doesn't match the component name 1:1.